src/redux/Books/bookSlice.jsx

```diff
@@ -1,21 +1,57 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  books: [],
+  books: [
+    {
+      item_id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
 };
 
 const bookSlice = createSlice({
   name: 'books',
   initialState,
 
   reducers: {
-    addBook: (state, action) => ({
-      ...state,
-      books: [...state, action.payload],
-    }),
+    // addBook: (state, action) => {
+    //   state.books = [
+    //     ...state.books,
+    //     {
+    //       id: Math.floor(Math.random() * 100),
+    //       title: action.payload.title,
+    //       author: action.payload.author,
+    //     },
+    //   ];
+    // },
```

Comment:
Please do not leave behind commented pieces of code in your codebase in order to not clutter it and to follow best practices.